### PR TITLE
#1491 NOTES - Appeals docket number is global

### PR DESCRIPTION
**Pull request by courtrightd**
Added docket number to the initial dialog, and error handling to require the user enter the number for most scenarios on the script. All scenarios now case note the docket number.

### CODE REVIEW

File: notes/appeals.vbs [hunk 1]
```diff
@@ -90,6 +92,8 @@ Do
 		Dialog Dialog1
 		cancel_without_confirmation
       	Call validate_MAXIS_case_number(err_msg, "*")
+        IF docket_number = "" then err_msg = err_msg & vbNewLine & "* Please enter a docket number or enter N/A if unknown."
+        IF appeal_actions <> "Received" AND appeal_actions <> "Summary Completed" AND isnumeric(docket_number) = False then err_msg = err_msg & vbNewLine & "* You have selected an action that should correspond to an existing docket number. Please enter a numeric docket number."
```
Comment (IlseFerris) [inline notes/appeals.vbs:96]: Line 96 - I think this can be removed now since docket number is now universal.

**Review by IlseFerris — CHANGES_REQUESTED** [review]
Ok just the one change, and can you add a changelog? Then I approve thee!

**Comment by courtrightd** [comment]
Added changelog.

File: notes/appeals.vbs (hunk 1 above)
Comment (courtrightd) [reply notes/appeals.vbs]: This is needed to allow for script actions that do not require docket number.

**Pull request merged**